addons/userconfig/cba_settings.sqf

```diff
@@ -16,18 +16,31 @@ force ace_finger_enabled = true;
 
 force ace_gforces_enabledFor = 0;
 
-force ace_goggles_effects = 0;
-
 force ace_hearing_enableCombatDeafness = false;
 
 force ace_map_mapShowCursorCoordinates = true;
 
-force ace_medical_ai_enabledFor = 0;                      // Medical system not enabled for AI
-force ace_medical_feedback_painEffectType = 2;            // Pain effect type is "Chromatic Abberation".
-force ace_medical_gui_enableMedicalMenu = 1;              // Enable medical menu.
-force ace_medical_playerDamageThreshold = 2;              // The amount of damage a player unit can take before dying.
+force ace_medical_AIDamageThreshold = 0.8;                // The amount of damage an AI unit can take before dying.
+force ace_medical_bleedingCoefficient = 1.2;              // Coefficient for controlling the bleeding speed
+force ace_medical_blood_maxBloodObjects = 300;            // Maximum number of blood drop objects that can be spawned
+force ace_medical_fatalDamageSource = 2;                  // Either large hits to vital organs or sum of trauma can be fatal
+force ace_medical_feedback_bloodVolumeEffectType = 2;     // Low blood volume effect is Icon + Color Fading
+force ace_medical_fractureChance = 0.3;                   // The probability of a fracture causing wound resulting in a fracture
+force ace_medical_ivFlowRate = 3;                         // IV Transfusion Flow Rate
+force ace_medical_painCoefficient = 1.4;                  // Coefficient for controlling the intensity of pain adjustments
+force ace_medical_playerDamageThreshold = 0.8;            // The amount of damage a player unit can take before dying.
+force ace_medical_spontaneousWakeUpChance = 0.15;         // Unconscious Wake Up Chance
+force ace_medical_spontaneousWakeUpEpinephrineBoost = 9;  // Epinephrine Wake Up Chance Boost
 force ace_medical_statemachine_AIUnconsciousness = false; // AI will not go unconscious for any reason, and therefore never enter cardiac arrest.
-force ace_medical_statemachine_fatalInjuriesAI = 0;       // AI will always die instantly.
```

Comment:
Add back key

Reply:
This is the default value, no need to define it here

Reply:
It's good to have for when someone decies to change it :)

And as reference is someone wants to look at our options